en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md

```diff
@@ -1,361 +1,104 @@
 
-## API-M Components
-
-WSO2 API Manager uses the following main components:
+## API-M Profiles
 
-<a href="{{base_path}}/assets/img/setup-and-install/api-m-components.png"><img src="{{base_path}}/assets/img/setup-and-install/api-m-components.png" alt="api-m components"></a>
+Listed below are the different profiles available in WSO2 API Manager. 
 
 <table>
     <tr>
         <th>
-            Component
-        </th>
+            Profile
+        </td>
         <th>
             Description
-        </th>
-    </tr>
-    <tr>
-        <td>
-            Gateway
-        </td>
-        <td>
-            Responsible for securing, protecting, managing, and scaling API calls.
         </td>
     </tr>
     <tr>
         <td>
-            Key Manager
+            Gateway Profile
         </td>
         <td>
-            Responsible for all security and key-related operations.
+            <p>Only starts the components related to the API Gateway.</p>
+<p>Use this when the API Gateway acts as a worker node in a cluster. This profile starts the backend features for data processing and communicates with the management node.</p>
         </td>
     </tr>
     <tr>
         <td>
-            Traffic Manager
+            Control Plane Profile
         </td>
         <td>
-            Used to make a decision on throttling. It also works as an event hub for broadcasting controller events such as throttling events, block conditions, revoke token retrieval events, API events, API policy events, application events, application policy events, application keys events, subscription events, and subscription policy events.
+            Starts all the API-M components (Traffic Manager, Key Manager, Publisher, Developer Portal) excluding the Gateway.
         </td>
     </tr>
     <tr>
         <td>
-            Publisher
+            Traffic Manager Profile
         </td>
         <td>
-            Enables API providers to easily publish their APIs, share documentation, provision API keys, and gather feedback on API features, quality, and usage.
-        </td>
-    </tr>
-    <tr>
-        <td>
-            Developer Portal
-        </td>
-        <td>
-            Enables consumers to self-register, discover API functionality, subscribe to APIs, evaluate them, and interact with API publishers.
+            Only starts the Traffic Manager component.
         </td>
     </tr>
 </table>
 
-## Databases for API-M components
+## API-M Components
 
-WSO2 API Manager components use the databases given below. Find out more about [databases in API-M]({{base_path}}/install-and-setup/setup/setting-up-databases/overview).
+Listed below are the five components of the API-M runtime and its function. 
 
```

Comment:
Add a statement similar to below here to specify the relationship of the profiles with componets, in a distributed setup. 
"When you run API-M instances in the recommended profiles, below components will be available in the rutime instances appropriately, that will operate the related functionalities as expected by each profile."